### ISSUE TITLE
Remove observing api key from package.json

### ISSUE THREAD

**Issue by Skiggz** [observing/fullcontact#31]
The observing account's api key was found in package.json here:

https://github.com/observing/fullcontact/blob/master/package.json#L14

This should perhaps be placeholder text the readme specifies to change, or an environment variable for the script to reference per user rather than sharing a key from your account in the open source library. That key has been disabled for that account's security in case it was by accident.

If you do however want to share a key, which I'd advise against, that key has been disabled so you'll need to update it to the new one in your account, but again I'd suggest just instructing users to use their own as any kind of weirdness from keys could trigger it being flagged or disabled in the future again anyways. 

cc @3rd-Eden

**Comment by AdriVanHoudt** [comment]
I think this was intentional but probably better off using travis secrets or something indeed

**Comment by Skiggz** [comment]
Yeah at least so somebody doesn't decide to use it, get it flagged, and then it doesn't work for anyone.
